operations/tasks/update-node_selector-labels.yml

```diff
@@ -0,0 +1,18 @@
+---
+# delete all predefined node selector labels
+- name: Delete all labels
+  command: >
+    oc label node {{ inventory_hostname }} {{ item }}-
+  with_items: {{ osbs_all_node_selector_labels }}
```

Comment:
Ideally, we'd want to remove only the managed labels that we don't plan on adding on the next task.
How about:
```
with_items: "{{ osbs_all_node_selector_labels | difference(osbs_node_selector_labels) }}"
```
Apparently ansible supports [set theory filters](http://docs.ansible.com/ansible/latest/playbooks_filters.html#set-theory-filters)!

If so, we should also consider doing something to prevent a label from being added that is not in the known list of managed. In the "Add labels" task below:
```
with_items: "{{ osbs_node_selector_labels | intersect(osbs_all_node_selector_labels) }}"
```

Reply:
ok